firmware/steering/kia_soul_ps/steering_control_module.ino

```diff
@@ -721,6 +115,8 @@ void setup( )
 
     // debug log
     DEBUG_PRINT( "init: pass" );
+
+    test_code = test_pid_values( pid_params, current_ctrl_state );
```

Comment:
It's intended for run-time.  If the test doesn't pass because the user changed the PID values, then the user is not allowed to continue.